src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -213,7 +213,7 @@ class Request
         self::HEADER_X_FORWARDED_FOR => 'for',
         self::HEADER_X_FORWARDED_HOST => 'host',
         self::HEADER_X_FORWARDED_PROTO => 'proto',
-        self::HEADER_X_FORWARDED_PORT => 'host',
+        self::HEADER_X_FORWARDED_PORT => 'for',
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded the `host` part will contain the value of the initial request's `Host` header.